vic/drivers/cesm/bld/vic.buildnml.csh

```diff
@@ -0,0 +1,3 @@
+#! /bin/csh -f
+
+cd $RUNDIR
```

Comment:
CaseDocs just contains read-only copies of input files in the run directory.
It's a place to store copies, that's all.


On 9/5/15 7:47 PM, Joe Hamman wrote:
>
> In vic/drivers/cesm/bld/vic.buildnml.csh 
> <https://github.com/UW-Hydro/VIC/pull/265#discussion_r38816956>:
>
> > @@ -0,0 +1,3 @@
> > +#! /bin/csh -f
> > +
> > +cd $RUNDIR
>
> @apcraig <https://github.com/apcraig> - Thanks for the tips. I ended 
> up getting a bit fancier than I originally intended but I like what we 
> have now. Default namelist files are copied from the 
> |vic/drivers/cesm/bld| directory to |$CASE/BuildConf/vicconf| where 
> they are editable on a case by case basis. They are also copied, in a 
> stripped down format to the |$RUNDIR|. I am writing a 
> |vic.input_data_list| file and that seems to be working right because 
> other parts of the setup scripts are complaining that I haven't moved 
> all the inputs to Spirit yet.
>
> My only remaining question is, what do we want in |$CASE/CaseDocs|? 
> Read only copies of VIC namelists (there are two, sort of...)? It is 
> quite easy to do that.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/UW-Hydro/VIC/pull/265/files#r38816956>.
>